README.md

```diff
@@ -125,6 +102,70 @@ With respect to the `tags` configuration option, the tags that will be added are
 
 If you want a metric to be strictly host-local, you can tell Veneur not to forward it by including a `veneurlocalonly` tag in the metric packet, eg `foo:1|h|#veneurlocalonly`. This tag will not actually appear in DataDog; Veneur removes it.
 
+# Configuration
+
+Veneur expects to have a config file supplied via `-f PATH`. The include `example.yaml` outlines the options:
+
+* `api_hostname` - The Datadog API URL to post to. Probably `https://app.datadoghq.com`.
+* `metric_max_length` - How big a buffer to allocate for incoming metric lengths. Metrics longer than this will get truncated!
+* `flush_max_per_body` - how many metrics to include in each JSON body POSTed to Datadog. Veneur will POST multiple bodies in parallel if it goes over this limit. A value around 5k-10k is recommended; in practice we've seen Datadog reject bodies over about 195k.
+* `debug` - Should we output lots of debug info? :)
+* `hostname` - The hostname to be used with each metric sent. Defaults to `os.Hostname()`
+* `interval` - How often to flush. Something like 10s seems good.
```

Comment:
random thought: should we mention that this choice is fairly permanent? as we discussed with DD, intervals for existing metrics have to be changed manually when you change this number. Might be good to call out as a gotcha.